eng/jacoco-test-coverage/pom.xml

```diff
@@ -24,10 +24,14 @@
 
   <properties>
     <azure-core.version>1.0.0-preview.3</azure-core.version>
+    <azure-core-amqp.version>1.0.0-preview.3</azure-core-amqp.version>
+    <azure-core-management.version>1.0.0-preview.3</azure-core-management.version>
```

Comment:
I don't see these new property versions used in the dependencies below.

Reply:
Yeah, I got ahead of myself and backed some things out. I'll return to these soon. Thanks.